app/controllers/api/jobs_controller.rb

```diff
@@ -16,10 +25,53 @@ def info_for_output(jobs)
         url: job["url"],
         latitude: job["latitude"],
         longitude: job["longitude"],
-        key: job["jobkey"],
+        job_key: job["jobkey"],
         location: job["formattedLocationFull"],
         date_posted: job["formattedRelativeTime"]
       }
     end
   end
+
+  def job_params
+    params.require(:job).permit(:jobkeys)
```

Comment:
That is what the api docs said. The request can look up multiple jobs by job key so thats why it is plural on their end.